### PR TITLE
Added an integration test to mb references api

### PR DESCRIPTION
**Pull request by lamuria**
To Run the integration test you should provide a valid Meo Wallet API
KEY with access to mb references api:

  `TOKEN:"wejkqwhekjqwkje" lein test`

ref: https://github.com/weareswat/clj-meowallet/issues/3

### CODE REVIEW

**Comment by paulosilva86** [comment]
🆗

**Comment by pawel-krysiak** [comment]
🆗

File: README.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ With Leiningen/Boot:
 [clj-meowallet "0.0.1"]
 ```
 
+#### Tests
+
+* `lein test` - runs the test suite *
+* `script/autotest` -listen for file changes and is always running tests
+
+* To Run the integration test you should provide a valid Meo Wallet API KEY with access to mb references api.
+  `TOKEN:"wejkqwhekjqwkje" lein test`
```
Comment (donbonifacio) [inline README.md]: How about a more specific name to the `TOKEN`? Is this a json token? auth token? Something like `MEO_WALLET_TOKEN`.

Comment (donbonifacio) [inline README.md]: Isn't this public?

Comment (donbonifacio) [inline README.md]: You can add a `profiles.clj` with env specific stuff like I did on the main service.

Comment (lamuria) [inline README.md]: :P when I was writing this documentation I thought that you'll talk about this 💃

File: test/clj_meowallet/core_test.clj [hunk 2]
```diff
@@ -70,3 +72,45 @@
 
       (testing "method-fn"
         (is (not (nil? (:method-fn result)))))))
+
+(deftest generate-mb-ref-test
+  (if-not (env :token)
+    (println "Warning: No meo wallet api key on env")
```
Comment (donbonifacio) [inline test/clj_meowallet/core_test.clj]: I'd add: _ignoring test_

**Comment by donbonifacio** [comment]
🆗

File: src/clj_meowallet/core.clj [hunk 3]
```diff
@@ -14,15 +14,15 @@
 
 (defn add-headers
   [credentials http-ops]
-  (if-let [token (:token credentials)]
-    (assoc http-ops :headers {"Content-Type:" "application/json"
-                              "Authorization: WalletPT " token})
+  (if-let [token (:meo_wallet_api_key credentials)]
```
Comment (donbonifacio) [inline src/clj_meowallet/core.clj]: You can use `:meo-wallet-api-key`

Comment (lamuria) [inline src/clj_meowallet/core.clj]: @donbonifacio yep, I already changed

**Comment by donbonifacio** [comment]
🆗

**Pull request merged**